src/main/java/org/hyperledger/aries/api/present_proof/PresentProofProposalBuilder.java

```diff
@@ -29,12 +31,13 @@ public class PresentProofProposalBuilder {
      * @return simple {@link PresentProofProposal} without any predicates set
      */
     public static PresentProofProposal fromCredential(@NonNull String connectionId, @NonNull Credential cred) {
-        final Map<String, String> attrs = cred.getAttrs();
+        final List<CredentialAttributes> attrs = cred.getAttrs();
```

Comment:
Revert. This is the CredentialRecord not the CredentialExchangeRecord it will never have a mime-type